rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java

```diff
@@ -148,19 +150,76 @@ public static String addEscape(String colName, String dbType) {
      * @param escape the escape
      * @return
      */
-    public static String addEscape(String colName, Escape escape) {
+    private static String addEscape(String colName, String dbType, Escape escape) {
         if (colName == null || colName.isEmpty()) {
             return colName;
         }
         if (colName.charAt(0) == escape.value && colName.charAt(colName.length() - 1) == escape.value) {
             return colName;
         }
-        return String.format("%s%s%s", escape.value, colName, escape.value);
+
+        KeywordChecker keywordChecker = KeywordCheckerFactory.getKeywordChecker(dbType);
+        if (keywordChecker != null) {
+            boolean check = keywordChecker.check(colName);
+            if (!check) {
```

Comment:
if 'colName' comes from user sql and check is false, Is it possible to return 'colName' directly.